src/Draco.Compiler/Syntax/Lexer.cs

```diff
@@ -85,11 +85,11 @@ public IToken Lex()
             // Normal tokens can have trivia
             this.ParseLeadingTriviaList();
             var token = this.LexNormal();
-            this.ParseTrailingTriviaList();
+            if (token.Type != TokenType.InterpolationEnd)
```

Comment:
Let's inline statements for one-liner ifs. Rule of thumb: fits in the same line => write in the same line. If not, use braces.